cli/Makefile

```diff
@@ -0,0 +1,13 @@
+makefile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
+makefile_dir := $(patsubst %/,%,$(dir $(makefile_path)))
```

Comment:
Isn't that going to end up as just `$PWD` all the time?

Reply:
Correct, which is what we want.